Storage/SQL/SQLiteHistory.swift

```diff
@@ -621,6 +621,7 @@ extension SQLiteHistory: BrowserHistory {
                 historyFTS.rowid = hist.rowid
             WHERE historyFTS.title LIKE "%\(searchTerm)%" OR
                 historyFTS.url LIKE "%\(searchTerm)%"
+            ORDER BY local_modified DESC
```

Comment:
I think I'll need to touch base with you about this one.

Reply:
Sure we sync on it, this was a request from Search design review they wanted to see the latest visits first